features/step_definitions/user_steps.rb

```diff
@@ -256,6 +256,14 @@ def get_series_name(age, classname, name)
   step %{I should not see "#{title}"}
 end
 
+When /^I change my username to "([^\"]*)"/h do |new_name|
```

Comment:
ambiguous first argument; put parentheses or a space even after the operator<br>unknown regexp options: h